templates/kubernetes/docs/install-manual.md

```diff
@@ -0,0 +1,364 @@
+| Google     | gcp-integrator       | juju deploy cs:~containers/gcp-integrator       | [docs](https://jujucharms.com/u/containers/gcp-integrator/)       |
+| Local      | N/A                  | N/A                                             |                                                           |
+| OpenStack  | openstack-integrator | juju deploy cs:~containers/openstack-integrator | [docs](https://jujucharms.com/u/containers/openstack-integrator/) |
+| Rackspace  | openstack-integrator | juju deploy cs:~containers/openstack-integrator | [docs](https://jujucharms.com/u/containers/openstack-integrator/) |
+| vSphere    | vsphere-integrator   | juju deploy cs:~containers/vsphere-integrator   | [docs](https://jujucharms.com/u/containers/vsphere-integrator/ )  |
+
+The charm should be deployed and relationships established with both the
+`kubernetes-worker` and `kubernetes-master` charms. For example, in the case of
+AWS:
+
+```bash
+juju deploy cs:~containers/aws-integrator
+juju trust aws-integrator
+juju add-relation aws-integrator kubernetes-master
+juju add-relation aws-integrator kubernetes-worker
+```
+
+The `juju trust` command allows the aws-integrator to make use of the
+credentials stored by **Juju**.
+
+This demonstrates how the charm relates to the rest of the **CDK** bundle, but it is
+recommended to use the [overlay](#overlay) method for installing in practice.
+
+## Customising the bundle install
+
+A number of the scenarios outlined at the start of this document involved
+customising  the **CDK** install. There are two main ways to do this, using
+overlays or editing the bundle file itself.
+
+Using an overlay means you can easily apply your customisation to different
+versions of the **CDK** bundle, with the possible downside that changes in the
+structure of  new versions of **CDK** may render your overlay obsolete or
+non-functional (depending on what exactly your overlay does).
+
+Saving a copy of the **CDK** bundle file and editing that means that your
+customisation will always work, but of course, requires that you create a new
+file for each version of **CDK**.
+
+Both methods are described below.
+
+<a id="overlay"></a>
+### Using overlays
+
+A _bundle overlay_ is a fragment of valid YAML which is dynamically merged on
+top of a bundle before deployment, rather like a patch file. The fragment can
+contain any additional or alternative YAML which is intelligible to **Juju**. For
+example, to replicate the steps used above to deploy and connect the
+`aws-integrator` charm, the following fragment could be used:
+
+```yaml
+applications:
+  aws-integrator:
+    charm: cs:~containers/aws-integrator
+    num_units: 1
+relations:
+  - ['aws-integrator', 'kubernetes-master']
+  - ['aws-integrator', 'kubernetes-worker']
+  ```
+
+You can also [download the fragment here][asset-aws-overlay]
```

Comment:
The full stop is missing at the end of this sentence.

Reply:
:+1: